app/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,93 @@
+<string name="error_field_required">حقل مطلوب</string>
+<string name="title_activity_maps">خريطة</string>
+<string name="login_warning_title">تحذير:</string>
+<string name="request_credentials_title">Pokémon Go أدخل اسم المستخدم وكلمة المرور الخاصة بك</string>
+<string name="search_pokemon_map_suggestion">الضغط لفترة طويلة على الخريطة لتغيير موقفها البحث</string>
+<string name="default_string">الافتراضي</string>
+<string name="pokemon_to_show">بوكيمون لعرض (فلتر)</string>
+<string name="action_relogin">إعادة الدخول</string>
+<string name="ban">منعت</string>
+<string name="minutes">"دقائق"</string>
+<string name="pokemon_disappearing_in">"تختفي في:"</string>
+<string name="pokemon_expired">منتهية الصلاحية</string>
+<string name="position_picked">الموضع المحدد</string>
+<string name="seconds">"s"</string>
+<string name="toast_localization_waiting">انتظار الموقع</string>
+<string name="toast_login_successful">تسجيل الدخول ناجح</string>
+<string name="toast_map_not_initialized">مشكلة تهيئة خريطة جوجل</string>
+<string name="toast_server_unreachable">غير قادر على الاتصال بخدمة بوكيمون</string>
+<string name="toast_token_expired">انتهت صلاحية الرمز المميز لتسجيل الدخول. الحصول على واحدة جديدة</string>
+<string name="toast_user_found">وجدت لك!</string>
+<string name="toast_login_error">فشل في تسجيل الدخول</string>
+<string name="toast_internal_error">حدث خطأ داخلي. قد يحدث هذا عندما تكون غير متصل</string>
+<string name="toast_ptc_login_error">PTC فشل تسجيل الدخول</string>
+<string name="toast_google_login_error">Google فشل تسجيل الدخول</string>
+<string name="enable_location_permission_title">تفعيل إذن الموقع</string>
+<string name="enable_location_permission_message">يرجى تمكين إذن الموقع لاستخدام هذا التطبيق</string>
+<string name="toast_no_location">فشل في البحث عن الاحداثيات</string>
+<string name="expiring_in">  %1$d:%2$02d : ينتهي في</string>
+<string name="google_login_failed">Google login فشل تسجيل الدخول</string>
+<string name="permition_message">يرجى تمكين إذن الموقع لاستخدام هذا التطبيق</string>
+<string name="allow">سماح</string>
+<string name="app_settings">إعدادات التطبيق</string>
+<string name="disclaimer">إخلاء المسؤولية</string>
+<string name="toast_searching">بحث</string>
+<string name="toast_pokemon_found_count">بوكيمون وجد</string>
+<string name="pokemon_found_none">لم يعثر على أي بوكيمون جديد للقبض </string>
+<string name="pokemon_found_new">%1$d بوكيمون جديد للقبض </string>
+<string name="notification_service_title">Pokémap خدمة</string>
+<string name="notification_service_scanning">فحص</string>
+<string name="notification_service_stop">Stop خدمة</string>
+<string name="notification_service_pokemon_near">%1$d بوكيمون قريب</string>
+<string name="notification_service_pokemon_in_area">%1$d بوكيمون في المنطقة</string>
+<string name="meters">متر</string>
+<string name="gym">Gym</string>
+<string name="button_ok">Ok</string>
+<string name="notification_service_inbox_line">%1$s (%2$d دقائق, %3$d متر)</string>
```

Comment:
Yep. That should work. We will see soon enough in the translated version if a number shows up ;)

If it depends on me, this translation goes up in the next build!